### DISCUSSION TITLE
Creating READABLE single file TW from server version

### DISCUSSION THREAD

**Discussion by Ca1vino** [TiddlyWiki/TiddlyWiki5#5626] [Q&A]
I've been making great progress at learning and customization.

A question, though -- when I export the server wiki to a single file, it works okay but the resulting file is READ ONLY in some way -- the "edit tiddler" buttons have disappeared. And I need the single-file version to give others their own local copy... people who can't run a server or use my servers.

I've searched documentation and the internet, all the results are about how to make the file read-only. But what I need is for it to be readable.

To be more specific, I'm reading the saved file off the local disk in the Chrome browser.

Is this doable?  Is there a setting for it, or do I need to serve the file rather than read it off local disk in the browser?

Thanks!

**Comment by saqimtiaz** [comment]
How are you exporting the wiki to the single file version?

**Reply by Ca1vino** [reply]
Good question. 

I got a big red error box when exporting from "Advanced Search". The error was:
"Uncaught TypeError: Cannot set property parentNode of #<Node> which has only a getter"

So I went to the folder of the node wiki and did this server command:
tiddlywiki ..\NotesWiki --rendertiddler "$:/core/save/all" Notes.html text/plain

The Notes.html file mostly worked when I opened it in Chrome. However, edit tiddler buttons are missing. It can create new tiddlers. There's a "Cloud" icon in the sidebar that turns red but doesn't succeed at doing anything.

So I guess something needs to be done to configure it as stand-alone?

If it needs changes, it would be a Good Thing to know how to script that.

**Reply by saqimtiaz** [reply]
Try making your offline copy with `tiddlywiki ..\NotesWiki --build index`
The command you have used includes the server sync plugins, and that is probably also why editing is disabled.

**Reply by Ca1vino** [reply]
Thanks!
Using "$:/plugins/tiddlywiki/tiddlyweb/save/offline" instead of "$:/core/save/all" did the trick.
